src/System.Private.Uri/src/System/Uri.cs

```diff
@@ -1897,18 +1897,9 @@ private static bool CheckForColonInFirstPathSegment(string uriString)
             return (index >= 0 && uriString[index] == ':');
         }
 
-        internal static unsafe string InternalEscapeString(string rawString)
-        {
-            if ((object)rawString == null)
-                return string.Empty;
-
-            int position = 0;
-            char[]? dest = UriHelper.EscapeString(rawString, 0, rawString.Length, null, ref position, true, '?', '#', '%');
-            if ((object?)dest == null)
-                return rawString;
-
-            return new string(dest, 0, position);
-        }
+        internal static unsafe string InternalEscapeString(string rawString) =>
```

Comment:
`unsafe` ?

Reply:
I didn't delete any of them but will.  Thanks.